Core/include/AnalysisTypes.h

```diff
@@ -69,8 +69,10 @@ enum class UncertaintySource {
     MuFakingTauES_DM1 = 114, MuFakingTauES_3prong = 115, TauVSmuSF_etaLt0p4 = 116, TauVSmuSF_eta0p4to0p8 = 117,
     TauVSmuSF_eta0p8to1p2 = 118, TauVSmuSF_eta1p2to1p7 = 119, TauVSmuSF_etaGt1p7 = 120, MuFakingTauES = 121,
     EleIdIsoUnc = 122, MuonIdIsoUnc = 123, TauTriggerUnc_DM0 = 124, TauTriggerUnc_DM1 = 125, TauTriggerUnc_DM10 = 126,
-    TauTriggerUnc_DM11 = 127, L1_prefiring = 128, PileUp = 129, PileUpJetId_eff  = 130, PileUpJetId_mistag  = 131, TauCustomSF_DM0 = 132,
-    TauCustomSF_DM1 = 133, TauCustomSF_DM10 = 134, TauCustomSF_DM11 = 135
+    TauTriggerUnc_DM11 = 127, L1_prefiring = 128, PileUp = 129, PileUpJetId_eff  = 130, PileUpJetId_mistag  = 131,
+    TauCustomSF_DM0 = 132, TauCustomSF_DM1 = 133, TauCustomSF_DM10 = 134, TauCustomSF_DM11 = 135,
+    VBFTauTriggerUnc = 136, lf = 137, hf = 138, hfstats1 = 139, hfstats2 = 140, lfstats1 = 141, lfstats2 = 142,
```

Comment:
- for VBF trigger there are 4 uncertainties:
   - 3 for Tau - for DM0, DM1 and DM10&11
   - 1 for jet legs
- Name btag uncertaties as `btag_lf` instead `lf` etc.